src/client.ts

```diff
@@ -355,6 +390,69 @@ export class Client {
     }
   }
 
+  /**
+   * This Method allows the user to take a screenshot. If no selector is passed
+   * then a full page screenshot is saved.
+   * if no filename is passed, then file is stored with a timestamp as the filename
+   * @param params filename, selector
+   * @returns Filename
+   */
```

Comment:
```
/**
 * Take a screenshot of the page and save it to `filename`.
 * If `selector` is passed in, it will take a screenshot of only that element
 * and its children as opposed to the whole page
 * 
 * @param params - Options
 * @param params.filename - The path of where to save the screenshot to
 * @param params.selector - Screenshot the given selector instead of the full page. Optional
 */
```